src/markdown/rehype-embed.ts

```diff
@@ -20,8 +20,7 @@ export const rehypeEmbed: Plugin<
         !parent ||
         !("tagName" in parent) ||
         parent.tagName !== "p" ||
-        parent.children.length > 1 ||
-        (node.children?.[0] as any)?.value !== node.properties?.href
```

Comment:
This judgment is to prevent rendering of such content:

```
![xLog](https://github.com/Crossbell-Box/xLog)
```

Because in this case, it is very likely that the author did not intend to render a card, so it is best to limit the scope to a pure link.

Reply:
Sorry, I misunderstand